conflictmonitor/mongo/a_init_replicas.js

```diff
@@ -1,9 +1,10 @@
+
 /*
 
 This script is the first of two scripts responsible for setting up mongoDB on initial boot. 
 These scripts should be copied to the /docker-entrypoint-initdb.d/ directory within the docker image;
 the docker image will then execute these scripts automatically when the database is first created. 
-This script and its partner are prefixed with the letters a and b respectively to ensure they are run 
+This script and is partner are prefixed with the letters a and b respectively to ensure they are run 
```

Comment:
This may have been an accidental change or maybe it wasn't complete. Probably should fix this grammatically.

Reply:
I replaced these files with new files, not sure why this changed